src/main/java/com/icthh/xm/ms/entity/service/KafkaTemplateDelegate.java

```diff
@@ -0,0 +1,43 @@
+package com.icthh.xm.ms.entity.service;
+
+import com.icthh.xm.commons.logging.LoggingAspectConfig;
+import lombok.RequiredArgsConstructor;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.support.SendResult;
+import org.springframework.stereotype.Service;
+import org.springframework.util.concurrent.ListenableFuture;
+
+@Service
+@LoggingAspectConfig(resultDetails = false)
+@RequiredArgsConstructor
+public class KafkaTemplateDelegate {
```

Comment:
Suggest to rename to `KafkaTemplateService`

Reply:
+